.travis.yml

```diff
@@ -1,33 +1,13 @@
 language: java
-env:
-  global:
-  - secure: STW9FkS69BRtMGlF+BGJEefaHi1UzguzkyQOZZxKZp93bJTzWafGLonwDW/Aky3BhlRwCfKrl9e0BuF+j1ACBv5gxowvO8RuwFzBNbpKOeTAtVEmqXrqfLtfN+w3tKj41Db08NxP9NNlxQst9FSlhJP0EM1EdetcMTYwMWvL2Q71UP2RD3vziorPON6W+hiXA1pstRHkYc3G5x7dmVvP6nMcUo/vPvbPi6QOn4SS2Pm/f9t5vAA1UYsXgfQDXG178wI9WMpiUo75hCHL2yzYnLszigWEzG2UDkAM35vJnLwI/fGhl4jysf8r85+bDusfkJ7TYQizRChLpfdOO67UwDTGAN28/KEvhSXT8TG6lMLu35xk3kRvN6mjcnYtxWYhaJjuXQ23vrdXfZvsoXbaW7DwPafqsG8nvhjMUXnXHmNUV+K4RIEqrnYmapxPTVAEmWluzyfCCEptpwpb0Xl2fc/KVy02o63a8rKaa2CaepZt6b7juFVTXeWYmaJ66NoHovXYtbGpiC2v1iDa8bmrdRkb1GzQ/PEcUTToGOeTOEkOZbvCMt2xS2B/cvgo3w2Z4C8moovY2dDjOP5F94fwBwbwodK/b5f8QBPwECqZIoQpqhvtMo99HOQJgpPhKLddUqhzw1rjPt07S2QDiHkwd4EePeZ6ZIcY61iGtMcwpes=
-  - secure: mesHrCi3fqaM538dEQJftOarOTEAMgo61uFx0RmOvg3nBVJphaLHoJWaYXfA5qtNhXTSYFwAeqIxAywRljEspRVwvhXvykRuGUzgk1w5MIL8OLflK34SU7m/eBPGVw6Ka4Uft8vnnZYzfl1qEY05EP9M20gciAujmfSnzSjfyIM9hZieDlsz4YkPexnuV9t7cFWEipg4icac6n0NFcZxzznnQ7Tyg+GyJA3HCVgB4jpfIPByJumpGBla1hFkfcWsgyL2uzQozcWUqLzW772PiPc6H75AAGSIDC63dwA3rmzCfBdj1POwn1mezRoJrxt0c3vr1uoSztZe5wquGEoUw1QRxOyEiYWy0z+TFWfb5G5D/H+7ICllL4+PXqI3Mf5uyujI3JNxeTbOa+lkydx79kL+ulOy2AyrY6IMmzRsyGmRxhpdoreQwHFKIsydqxEtBXHZ3oCcjR83hxLF9atbefWq8CNfvdLYsUUxwsUSmF+IBo5se/kJ7zwdb9yFWX2NzlJhchDmU97dXIzdn8PI4mFC0p/gmllJvr/cdjSBxLxgPVcH8fQ0kH6oMgNEXKvl1od/Qls95w0nP4W33zYxEk+DCKwdCXIQ8arEj4lamSEGyzii0zdn1gM/LExyu/SQ8ZOeWTEOLHuWYMNj7zuEVmaf1b1wwZf8a5E8hH4W6RA=
 sudo: false
 branches:
```

Comment:
why did we leave this?

Reply:
I left the travis.yml temporarily because I still wanted to keep automated builds working in Travis until GitHub Actions gets put in place. But if we're ready to merge it in, I can remove it.